plugins/GSdx/GS.h

```diff
@@ -1425,3 +1425,25 @@ enum class Filtering : uint8
 	Trilinear_Bilinear_Forced,
 	Trilinear_Always
 };
+
+//Enum order matters! Don't change it!
+enum class CRCHackLevel : int8
+{
+	Automatic = -1,
+	None = 0,
+	Minimum = 1,
+	Partial = 2,
+	Full = 3,
+	Aggressive = 4
```

Comment:
FYI, initializing the first enum element is enough and the succeeding elements value will follow arithmetic progression with a difference of 1.